### ISSUE TITLE
Problem with PHP 8.0 Constructor property promotion

### ISSUE THREAD

**Issue by stanislav-janu** [nette/coding-standard#42]
I run `temp/nette-coding-standard/ecs check app tests --preset php74`.

Result
```
1) app/AdminModule/forms/SignInFormFactory.php
    ---------- begin diff ----------
--- Original
+++ New
@@ -12,8 +12,10 @@
 {
 	use Nette\SmartObject;
+	public function __construct(private FormFactory $factory;
-	public function __construct(private FormFactory $factory, private User $user)
+
+    public function __construct(private FormFactory private User $user)
 	{
 	}
@@ -47,3 +49,4 @@
 		return $form;
 	}
 }
+,,
\ No newline at end of file
    ----------- end diff -----------
Applied checkers:
 * Nette\CodingStandard\Sniffs\WhiteSpace\FunctionSpacingSniff.After
 * Nette\CodingStandard\Sniffs\WhiteSpace\FunctionSpacingSniff.Before
 * SlevomatCodingStandard\Sniffs\Classes\DisallowMultiPropertyDefinitionSniff.DisallowedMultiPropertyDefinition
 * SlevomatCodingStandard\Sniffs\Classes\TraitUseSpacingSniff.IncorrectLinesCountAfterLastUse
 * SlevomatCodingStandard\Sniffs\Functions\TrailingCommaInCallSniff.MissingTrailingComma
 ```

Maybe is problem, because preset is for 7.4. But there is no preset for 8.0. I would write 8.0 preset, but I don't know how. :-)

**Comment by dg** [comment]
This repository is only a package of presets for symplify/easy-coding-standard, friendsofphp/php-cs-fixer, slevomat/coding-standard. You must report the issues to the appropriate package.
